com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -105,6 +105,7 @@ float4x4 unity_ObjectToWorld;
 float4x4 unity_WorldToObject;
 float4 unity_LODFade; // x is the fade value ranging within [0,1]. y is x quantized into 16 levels
 real4 unity_WorldTransformParams; // w is usually 1.0, or -1.0 for odd-negative scale transforms
+float4 unity_RenderingLayer;
```

Comment:
Since this is a different block, we should add an empty line and a comment before it
```suggestion

// Render Layer block feature
float4 unity_RenderingLayer;
```

Reply:
Please document what's in the 4 components

Reply:
This is verbatim copy of HDRP code&layout. I will add comment and change layout.

Reply:
Updated.